js/configs.js

```diff
@@ -1,17 +1,16 @@
 export default {
   appname: "overpass-turbo",
-  defaultServer: "//overpass-api.de/api/",
+  defaultServer: "https://overpass-api.de/api/",
   suggestedServers: [
-    "//overpass-api.de/api/",
+    "https://overpass-api.de/api/",
     "https://overpass.kumi.systems/api/",
-    "http://overpass.openstreetmap.ru/cgi/",
-    "//overpass.openstreetmap.fr/api/"
+    "https://overpass.openstreetmap.ru/cgi/"
   ],
   defaultTiles: "//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
```

Comment:
```suggestion
  defaultTiles: "https://tile.openstreetmap.org/{z}/{x}/{y}.png",
```
and perhaps even for opencyclemap which are commented